types/time.go

```diff
@@ -840,13 +840,14 @@ func (d Duration) ToNumber() *MyDecimal {
 // ConvertToTime converts duration to Time.
 // Tp is TypeDatetime, TypeTimestamp and TypeDate.
 func (d Duration) ConvertToTime(tp uint8) (Time, error) {
-	year, month, day := gotime.Now().Date()
-	// just use current year, month and day.
-	n := gotime.Date(year, month, day, 0, 0, 0, 0, gotime.Local)
```

Comment:
why not just `gotime.Date(year, month, day, 0, 0, 0, 0, gotime.UTC)` ?

Reply:
Use `mixDateAndTime` instead of Go time library make it more MySQL compatible.